local/item.go

```diff
@@ -70,6 +72,48 @@ func (i *item) Open() (io.ReadCloser, error) {
 	return os.Open(i.path)
 }
 
+// LimitedReadCloser wraps io.LimitedReader and exposes the Close() method.
+type LimitedReadCloser struct {
```

Comment:
It is enough to include `io.ReadCloser` - don't need to specify `io.Reader` too.